app/models/intake.rb

```diff
@@ -122,6 +122,10 @@
 #  needs_help_2019                                      :integer          default(0), not null
 #  needs_help_2020                                      :integer          default(0), not null
 #  needs_help_2021                                      :integer          default(0), not null
+#  needs_help_current_year                              :integer
```

Comment:
`needs_help_2018` etc are non-null; it'd be good to match that pattern. You'll need to `rails db:rollback`, change the migration, then `rails db:migrate`.